sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/PartCreationPolicyShouldBeUsedWithExportAttribute.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.ComponentModel.Composition;
+
+namespace Tests.Diagnostics
```

Comment:
Please add tests with other `CreationPolicy` values.

Reply:
Isn't the CreationPolicy irrelevant?

Reply:
It's irrelevant hence adding a test to show it is!

Reply:
There are many irrelevant things for this rule :)